test/acquisition/test_proximal.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+from botorch.acquisition.analytic import ExpectedImprovement
+from botorch.acquisition.monte_carlo import qExpectedImprovement
+from botorch.acquisition.proximal import ProximalAcquisitionFunction
+from botorch.models import SingleTaskGP
+from botorch.utils.testing import BotorchTestCase
+from torch.distributions.multivariate_normal import MultivariateNormal
+
+
+class TestProximalAcquisitionFunction(BotorchTestCase):
+    def test_proximal(self):
+        train_X = torch.rand(5, 3, device=self.device)
```

Comment:
Let's wrap these tests in a `for dtype in (torch.float, torch.double):` just for good measure. I don't see anything that would break here but it helps avoid minor bugs.